src/pages/appflow/destinations/google.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Google Play store'
+previousText: 'Deploy To Apple Appstore'
+previousUrl: '/docs/appflow/destinations/apple'
+nextText: 'Builds/Automations'
+nextUrl: '/docs/appflow/destinations/builds'
+---
+
+The Google Play destination allows you to upload your finished binaries to the store for Android.
+
+![/docs/assets/img/appflow/google-overview.png](/docs/assets/img/appflow/google-overview.png)
+
```

Comment:
I think it would be good to put in a Warning box here that calls out the gotcha that the first upload to the play store must be done in the play console. warning box done using `<blockquote>` tags like here https://github.com/ionic-team/ionic-docs/blame/master/src/pages/appflow/deploy/setup.md#L16